### ISSUE TITLE
Multiple port forward rules fail

### ISSUE THREAD

**Issue by isontheline** [isontheline/pro.webssh.net#740]
# Bug description
It seams if I create a tunnel with several port forwarding entries as bellow:

11022:srv-01.local:22
11122:srv-02.local:22
11222:srv-03.local:22

and try to connect, I don't see the active channel in the left hand side navigation. So there is no way to disconnect this channel. So only restart of tha app would release the port forwarding.

Also it seams that the very first port forwarding rule in the list doesn't work. Only the second and third would work.. 

From **Sergey** by *Email*

**Comment by isontheline** [comment]
A fix (already pushed) will be available in a few hours

**Comment by isontheline** [comment]
> hello, in my case the issue is still present. I did update the up from the applese store before testing.

> I have in the list Port Forwarding list:
> 21 entries.

> Perhaps that is the amount at entires causing the problem. I've also noticed that the tunnels are bing opened up in sequence not in parallel, so I had to wait about 15-20 seconds before I could connect.

> One more thing, If I have a list in the Port Forwarding, the very first rool doesn't work, so the very first entlry I always need to duplicate in thelist. 

> Hope to hear good news soon from your team.

From **Sergey**

**Comment by isontheline** [comment]
I'm closing this in favor of #747
